detect.go

```diff
@@ -47,6 +49,13 @@ func Detect(gemfileParser VersionParser) packit.DetectFunc {
 		var versionSource string
 		if mriVersion != "" {
 			versionSource = "Gemfile"
+		} else {
+			// fall back to .ruby-version file
+			rubyVersion, err := rubyVersionFileParser.ParseVersion(rubyVersionFile)
+			if err == nil && rubyVersion != "" {
```

Comment:
As discussed, if the validation of `.ruby-version` fails, we don't error out and just don't set the `mriVersion`. But I feel like we should still inform the user that we tried to parse it and something went wrong. Can we emit a log warning?

Reply:
I think so!

Reply:
Something like `logger.Subprocess("WARNING: Could not parse the .ruby-version file, as a result no Ruby version has been specified")`

Reply:
I added this warning 👍